cppSlint/.vscode/settings.json

```diff
@@ -0,0 +1,19 @@
+{
+  "torizon_psswd": "",
+  "torizon_login": "",
+  "torizon_ip": "",
+  "host_ip": "",
+  "torizon_workspace": "${workspaceFolder}",
+  "torizon_debug_port": "",
+  "torizon_debug_ssh_port": "2231",
+  "torizon_debug_port2": "2232",
+  "torizon_debug_port3": "",
+  "terminal.integrated.defaultProfile.linux": "bash",
+  "torizon_gpu": "",
+  "torizon_arch": "",
+  "wait_sync": "1",
+  "torizon_run_as": "torizon",
+  "torizon_app_root": "/home/torizon",
+  "tcb.packageName": "__change__",
+  "tcb.version": "early-access"
```

Comment:
Did you do not want to add here the:

```json
  "slint.preview.providedByEditor": true
```

I was testing here and this integrates very well instead of the default preview that uses a fullscreen window

Reply:
I'd prefer to stay with the defaults here. The side-by-side works, but in terms of screen real estate that remains left for the code editor, I prefer the separate preview window.

Reply:
Is there a way to not show the window in full screen, then? My concern is that it makes users, that uses single screen, confused.

Reply:
ok, I tried again and it's showing a normal window, resolving also this one